rep-2004.rst

```diff
@@ -434,86 +354,318 @@ For example, this might be a tutorial package which is not intended for reuse bu
 Package Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-*Note: bullets below that start with [ROS Core], will be the prescription for what we do in the core packages in order to meet the associated requirements*
-
 Requirements to be considered a 'Level 4' package:
 
-* Version Policy:
-
-  * No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
+#. **Version Policy:**
 
-* Change Control Process:
+   #. No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
```

Comment:
the `even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)` part of this can't be mapped to the chart, another reason to maybe keep the written requirements along with the chart.

Reply:
Isn't it just an empty spot in the `1.i` part of the chart?

Reply:
It's actually an empty spot for `1.ii` (`Must be at a stable version (e.g. for semver that means have a version >= 1.0.0)`), but I'm leaning towards leaving the written requirements and the chart anyway so it's not an issue either way